pandapower/timeseries/output_writer.py

```diff
@@ -332,6 +333,11 @@ def log_variable(self, table, variable, index=None, eval_function=None, eval_nam
             >>> ow.log_variable('res_line', 'loading_percent', index=[0, 2, 5]) # add logging for line loading of lines with indices 0, 2, 5
             >>> ow.log_variable('res_line', 'loading_percent', eval_function=pd.max) # get the highest line loading only
 
+            # Getting the cost function slope for each time step:
+            >>> def cost_logging(result, n_columns=2):
+            >>>      return array([result[i][0][2] for i in range(len(result))])
+            >>> ow.log_variable("pwl_cost", "points", eval_function=cost_logging)
+
```

Comment:
Could you provide some more descriptions / add something to the docs? I thinks it's not clear from the example that this is only for OPF time series calculations and what your cost_logging function does.

Reply:
This is not only for OPF time series, this could be applied to any random function. This was just the use case I had. This is for functions that return more than one value in general. Where do you suggest more documentation? I could extend the time_series_advanced_output.ipynb...?

Reply:
Yeah good idea. But it would be great also to explain it in the docstring. I think it's otherwise not obvious why that is beneficial. It will appear here: https://pandapower.readthedocs.io/en/v2.2.2/timeseries/output_writer.html